cmd/hyperkube/hyperkube.go

```diff
@@ -123,8 +123,8 @@ func (hk *HyperKube) Run(args []string) error {
 	command := args[0]
 	baseCommand := path.Base(command)
 	serverName := baseCommand
+	args = args[1:]
 	if serverName == hk.Name {
-		args = args[1:]
```

Comment:
As long as you run ``hyperkube kubectl``, this should not matter. Why movin 2 lines higher?